python/dolfinx/jit.py

```diff
@@ -90,35 +90,40 @@ def mpi_jit(*args, **kwargs):
 
 
 @mpi_jit_decorator
-def ffcx_jit(ufl_object, form_compiler_parameters=None):
+def ffcx_jit(ufl_object, form_compiler_parameters={}, jit_parameters={}):
     # Prepare form compiler parameters with overrides from dolfinx
     p = ffcx.default_parameters()
     p["scalar_type"] = "double complex" if common.has_petsc_complex else "double"
-    p.update(form_compiler_parameters or {})
+    p.update(form_compiler_parameters)
 
     # CFFI compiler options/flags
-    extra_compile_args = ['-g0', '-O3', '-march=native']
-    user_cflags = os.getenv('DOLFINX_JIT_CFLAGS')
-    if user_cflags is not None:
-        extra_compile_args = user_cflags.split(" ")
-    cffi_options = dict(cffi_extra_compile_args=extra_compile_args, cffi_verbose=False,
-                        cffi_debug=False)
+    jit_params = {"cffi_extra_compile_args": ["-O3", "-g0", "-march=native"],
+                  "cffi_debug": False, "cffi_verbose": False,
+                  "cffi_libraries": None, "cache_dir": "~/.cache/fenics", "timeout": 10}
```

Comment:
I know that was thee before, but we should really find a way to get the platform-dependent cache location.

Reply:
I'm not sure there is a way to get a platform independent cache directory. However, we could at least use `pathlib.Path.home()` to get the home directory, and then add `.cache/fenics` to that. It will then work on Windows :smiley: